File: 백준/Bronze/4562. No Brainer/No Brainer.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        
        for (int i = 0; i < n; i++) {
            int X = sc.nextInt();
            int Y = sc.nextInt();
            
            if (X < Y) {
                System.out.println("NO BRAINS");
            } else {
                System.out.println("MMM BRAINS");
            }
        }
    }
}

File: 백준/Bronze/4562. No Brainer/No Brainer.cs
using System;

class Q4562
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        
        for (int i = 0; i < n; i++)
        {
            string[] S = Console.ReadLine().Split();
            int X = int.Parse(S[0]);
            int Y = int.Parse(S[1]);
            
            if (X < Y)
            {
                Console.WriteLine("NO BRAINS");
            }
            else
            {
                Console.WriteLine("MMM BRAINS");
            }
        }
    }
}